api/http/handler/fdo/fdo.go

```diff
@@ -0,0 +1,79 @@
+package fdo
```

Comment:
didn't notice before, but this should go inside the `api/http/handler/hostmanagement` together with the `openamt` package